fetch.bs

```diff
@@ -4289,6 +4289,26 @@ steps:
 <var>response</var>, a <a for=/>global object</a> <var>global</var>, and a <a for=/>string</a>
 <var>initiatorType</var> (default "<code>other</code>"), run these steps:
 
+<ol>
+ <li><p>Let <var>taoPass</var> be true if <var>response</var>'s
+ <a for=response>timing allow passed flag</a> is set; otherwise false.
+
+ <li>Set <var>response</var>'s <a for="response">timing info</a> to the
+ result of calling <a>report timing</a> given <var>response</var>, <var>global</var>,
+ <var>initiatorType</var>, and <var>taoPass</var>.
+</ol>
+
+<p>To <dfn export>report timing with origin</dfn> given a <a for=/>response</a>
```

Comment:
with an origin*?

Reply:
This algorithm should also have a note that indicates it's called by HTML, ideally with a reference to the algorithm that does it.